kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchResultUtils.java

```diff
@@ -558,6 +582,43 @@ public static KaldbSearch.SearchRequest.SearchAggregation toSearchAggregationPro
     }
   }
 
+  public static Map<String, FiltersAggBuilder.FilterAgg> fromFiltersAggregation(
+      KaldbSearch.SearchRequest.SearchAggregation.FiltersAggregation searchFiltersAggregation) {
```

Comment:
This is due to the fact these are the protobuf definitions, and are using a nested message structure. They're not actually are not using fully qualified names - that would be `com.slack.kaldb.proto.service.KaldbSearch.SearchRequest.SearchAggregation.FiltersAggregation` instead of the marginally shorter `KaldbSearch.SearchRequest.SearchAggregation.FiltersAggregation`.

These could be shorter if we move away from the nested message structure and went with a flatter message structure. That's something outside the scope of this PR, and involves a tradeoff of making the protobuf definition harder to follow. If this is an approach we prefer however, we can certainly do that change.